src/card.ts

```diff
@@ -17,10 +17,11 @@ export interface Card {
   count: number;
 }
 
-const CARD_VALUES = Object.values(SignContent);
+// const CARD_VALUES = Object.values(SignContent);
 
-export function getInitialCards(): Card[] {
-  const date = new Date().toISOString();
+export function getInitialCards(countCardsInPlay: number): Card[] {
+    const date = new Date().toISOString();
+    const CARD_VALUES = Object.values(SignContent).slice(0, countCardsInPlay)
 
   const cards = [...CARD_VALUES, ...CARD_VALUES].map((value, index) => ({
```

Comment:
```suggestion
To randomize which cards get used each game:

```suggestion
  let cardValues = Object.values(SignContent);
  shuffleArray(cardValues);
  cardValues = cardValues.slice(0, countCardsInPlay);

  const cards = [...cardValues, ...cardValues].map((value, index) => ({
```